inc/notificationtargetcommonitilobject.class.php

```diff
@@ -1319,16 +1319,15 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
             $tmp['##followup.isprivate##']   = Dropdown::getYesNo($followup['is_private']);
 
             // Check if the author need to be anonymized
-            if (ITILFollowup::getById($followup['id'])->isFromSupportAgent()
-               && $is_self_service
+            $tmp['##followup.author##'] = getUserName($followup['users_id']);
```

Comment:
Should stay in `else`, as it was in original version..

Reply:
refactor done